Content.Server/RoundEnd/RoundEndSystem.cs

```diff
@@ -224,6 +235,12 @@ public void CancelRoundEndCountdown(EntityUid? requester = null, bool checkCoold
             _countdownTokenSource.Cancel();
             _countdownTokenSource = null;
 
+            _replayEventSystem.RecordReplayEvent(new ShuttleReplayEvent()
+            {
+                Source = requester == null ? null : _replayEventSystem.GetPlayerInfo(requester.Value),
+                Severity = ReplayEventSeverity.High,
+                EventType = ReplayEventType.EvacuationShuttleRecalled,
+            });
             if (requester != null)
```

Comment:
This should probably be a separate event type since the `Countdown` field isn't used here.